src/main/kotlin/com/sourcegraph/cody/agent/protocol_extensions/CodyTaskState.kt

```diff
@@ -0,0 +1,23 @@
+package com.sourcegraph.cody.agent.protocol_extensions
+
+object CodyTaskStateExt {
```

Comment:
I do not think we should be using cody task states anymore, do we really need this?